src/Compatibility/Microsoft.DotNet.PackageValidation/BaselinePackageValidator.cs

```diff
@@ -43,7 +46,7 @@ public void Validate(Package package)
                 ContentItem latestCompileTimeAsset = package.FindBestCompileAssetForFramework(baselineTargetFramework);
                 if (latestCompileTimeAsset == null)
                 {
-                    if (!_checker.Contain(DiagnosticIds.TargetFrameworkDropped, baselineTargetFramework.ToString()))
+                    if (!_diagnosticBag.Contain(DiagnosticIds.TargetFrameworkDropped, baselineTargetFramework.ToString()))
```

Comment:
why check for this instead of just logging the errors?

Reply:
I don't understand that, the diagnostic bag will already ignore the suppressed errors.